.github/workflows/cd.yaml

```diff
@@ -0,0 +1,45 @@
+name: Build and Upload to CDN
+on:
+  workflow_dispatch:
+    inputs:
+      commit:
+        description: 'Commit SHA'
+        required: false
+      version:
+        description: 'Version (should follow semver v1.2.3)'
+        required: false
+  push:
+    branches:
+      - master
+jobs:
+  test:
+    uses: ./.github/workflows/ci-go.yaml
+    secrets: inherit
+  build:
+    needs: test
+    runs-on: ubuntu-latest
```

Comment:
building only for ubuntu?

Reply:
for now it will suffice, but sure we can add it soon